tools/IoTEdgeDevOps/DevOpsLib/BugManagement.cs

```diff
@@ -2,15 +2,15 @@
 namespace DevOpsLib
```

Comment:
The diff here is a bit misleading. I renamed BugManagement.cs -> BugWiqlManagement.cs for clarity. Then I made a new file for creating the bugs called BugManagement.cs.

The diff here is between the wiql api file (left), which has not been changed but just moved, and the new file (right) which took on the same name as the wiql api file but is doing the bug creation.

Reply:
Are both files needed? Or would it be better/simpler to consolidate them into a single BugManagement file and class? I see that they both use different subsets of the overarching Work Item Tracking APIs and they both perform operations related to work items: One creates bugs and the other gets the bug count. It seems reasonable to have both of these operations fall under the same abstraction.

Reply:
The pattern is that each individual api has its own class. These apis share nothing other than the partial name "Bug", so I thought it best to keep following the pattern and have separate classes.